administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -188,10 +188,13 @@ COM_LOCALISE_CANNOT_REMOVE_DEFAULT_LANGUAGE="Localise cannot remove default lang
 
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT="Translate"
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT_DESC="Translate description"
+COM_LOCALISE_CHECKBOX_TRANSLATION_GRAMMAR_CASE="Grammar case"
```

Comment:
Is that one still used? I see also the code
```
if ($istranslation)
			{
				$title = Text::_('COM_LOCALISE_REVISED');
				$tip   = $title;
			}
			else
			{
				$title = Text::_('COM_LOCALISE_CHECKBOX_TRANSLATION_GRAMMAR_CASE');
				$tip   = Text::_('COM_LOCALISE_TOOLTIP_TRANSLATION_GRAMMAR_CASE');
			}
```

How to test?
If it is used, the English "Grammar case" is confusing.

Reply:
I found how to test.
The tip is almost unreadable which means it should be displayed in another way and we definitely should use `Modified` instead of the wording "Grammar case" as this is not applicable to the situation.
![Screen Shot 2021-10-04 at 12 05 16](https://user-images.githubusercontent.com/869724/135833012-db2f2f7d-358b-4ffd-9836-0cbd3de3fc7c.png)

Reply:
Nice idea, i will take a look.